cmd/eyes/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"os"
+
+	"github.com/tendermint/tmlibs/cli"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/cmd/basecoin/commands"
+	"github.com/tendermint/basecoin/modules/base"
+	"github.com/tendermint/basecoin/modules/etc"
+	"github.com/tendermint/basecoin/stack"
+)
+
+// BuildApp constructs the stack we want to use for this app
+func BuildApp() basecoin.Handler {
+	return stack.New(
+		base.Logger{},
+		stack.Recovery{},
+	).
```

Comment:
Is this set of commands for if you wanted to run an _eyes_ client out of process from basecoin? Can we run it in process? if yes, when do we even want out of process functionality?